Remove references to various ex MinIO Console features (#1373)
minio/docs: source/administration/console/managing-objects.rst

   If you enable versioning, you can specify prefixes to exclude from versioning.

> You **must** configure locking and versioning options at the time of bucket creation.

Reviewer: In context this works OK, but it's not as clear as it could be (I tripped over it until I expanded upward) ... if you feel like it, try something like

```suggestion
You can configure locking and versioning options only when you create the bucket.
```
